packages/ruleset-migrator/src/tree/scope.ts

```diff
@@ -0,0 +1,102 @@
+const REGISTERED_WORDS = [
```

Comment:
Is there a list/package of reserved words we can pull from so we don't have to maintain it here?

Reply:
Seems like https://www.npmjs.com/package/reserved, will use it, looks cool.